winevdm.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = otvdm
```

Comment:
It would be better to name it "otya128's WineVDM" (see the chat for details)

Reply:
Or maybe better just "WineVDM". As I know, usually we don't specify developer/company names in the app's names.